### PR TITLE
Modified toast

### PR DESCRIPTION
**Pull request by nameerakhter**
### Description:
This PR addresses the issue #12 

### Issue Fixed:
issue #12 

### Please check all the relevant boxes:
- [❌] Does this PR break UI anywhere?
- [✅ ] Have you tested the changes thoroughly before creating this PR?
- [❌ ] Have you pushed any sensitive information along with the changes?
- [❌] Is this issue related to any other issue?
- [❌] Does your change generate any new warnings?
- [❌] Have you added screenshots with the PR?

### CODE REVIEW

**Comment by nameerakhter** [comment]
@poswalsameer added some basic styles for now for toast messages let me know if these are OK or we can use something like react-toastify later that will give us a lot more customization over toast messages

File: client/campusburn/src/app/login/page.tsx [hunk 1]
```diff
@@ -58,12 +60,17 @@ export default function Page() {
...
                 title: "User not found",
-                variant: "destructive"
+                variant: "destructive",
+                className: "bg-red-600 text-white"
             });
         }
         else if( loginResponse.status === 200 ){
             console.log("Response after login: ", loginResponse);
-
+            toast({
```
Comment (poswalsameer) [inline client/campusburn/src/app/login/page.tsx:69]: This toast is not required on login successful, as we are already pushing the user to a new route.

**Comment by poswalsameer** [comment]
> @poswalsameer added some basic styles for now for toast messages let me know if these are OK or we can use something like react-toastify later that will give us a lot more customization over toast messages

ShadCN toasts are fine, if you want you can use [Sonner](https://ui.shadcn.com/docs/components/sonner) too.
Also, can you please attach the screenshots of the changes. That would help a lot.

File: client/campusburn/src/app/login/page.tsx (hunk 1 above)
Comment (nameerakhter) [reply client/campusburn/src/app/login/page.tsx]: Sure I'll update it

**Comment by nameerakhter** [comment]
@poswalsameer updated the changes as you suggested removed toast message on successful login

also here are some screenshots for the updated toast messages

updated success toast
Screenshot 2024-12-14 230306
updated error toast

Screenshot 2024-12-14 230416

**Comment by poswalsameer** [comment]
> @poswalsameer updated the changes as you suggested removed toast message on successful login
> 
> also here are some screenshots for the updated toast messages
> 
> updated success toast Screenshot 2024-12-14 230306 updated error toast
> 
> Screenshot 2024-12-14 230416

Can you prepare a design like the one attached below? Basically, we need to have minimal toasts with a gradient and transparent background.

Screenshot 2024-12-15 143242

**Comment by nameerakhter** [comment]
> > @poswalsameer updated the changes as you suggested removed toast message on successful login
> > also here are some screenshots for the updated toast messages
> > updated success toast Screenshot 2024-12-14 230306 updated error toast
> > Screenshot 2024-12-14 230416
> 
> Can you prepare a design like the one attached below? Basically, we need to have minimal toasts with a gradient and transparent background.
> 
> Screenshot 2024-12-15 143242

Sure I'll design it and update the PR also let me know if you want the same layout as well like on top you want to show a title of toast and then you want to add a description as well for toast

**Comment by nameerakhter** [comment]
@poswalsameer 

updated the Toast messages here are the screenshots let me know if these work

Screenshot 2024-12-15 155225

Screenshot 2024-12-15 155151

**Comment by poswalsameer** [comment]
> @poswalsameer
> 
> updated the Toast messages here are the screenshots I didn't exactly tried to replicate the design because then I would have to write custom CSS just tailwind might not be enough, let me know what you think
> 
> Screenshot 2024-12-15 155225
> 
> Screenshot 2024-12-15 155151

Cool, merging this now. Thanks buddy!

**Pull request merged**